src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -848,7 +867,7 @@ private void logConfiguration()
                value = dsProps;
             }
             value = (prop.contains("password") ? "<masked>" : value);
-            LOGGER.debug((prop + "................................................").substring(0, 32) + (value != null ? value : ""));
+            LOGGER.debug((prop + "................................................").substring(0, 32) + value);
```

Comment:
If that is the goal, this doesn't accomplish it in terms of debugging.  Properties with trailing spaces would still be hard to debug.
```java
value = (value instanceof String) ? (value != null ? '"' + value + '"' : value) : value;
```